remote-trait-object/src/port.rs

```diff
@@ -28,8 +29,7 @@ use std::sync::Arc;
 pub trait Port: Send + Sync + 'static {
     fn call(&self, packet: PacketView) -> Packet;
     fn delete_request(&self, id: ServiceObjectId);
-    /// TODO: Assign id automatically and return it.
-    fn register(&self, id: String, handle_to_register: Box<dyn Dispatch>);
+    fn register(&self, service: Arc<dyn Dispatch>) -> HandleToExchange;
```

Comment:
How about service_object?
```suggestion
    fn register(&self, service_object: Arc<dyn Dispatch>) -> HandleToExchange;
```